R/bbr.R

```diff
@@ -321,9 +321,41 @@ bbi_help <- function(.cmd_args=NULL) {
 #'   model. `submit_model()` and `submit_models()` also support passing a
 #'   configuration file via the `.config_path` argument.
 #'
+#'  **Notes on `.nonmem_dir`**
+#'
+#'   `bbi` tries to detect subdirectories for different NONMEM versions, along
+#'   with a license file and executable within the subdirectory.
```

Comment:
My preference would be to drop this entirely.  What `bbi` looks for to decide a directory is a NONMEM installation is an implementation detail, and I find it a bit confusing as a lead off paragraph.  For example

> `bbi` tries to detect subdirectories for different NONMEM versions

Where does it look for these?  The reader is supposed to infer `.nonmem_dir`, at this point they haven't even been given the details below about htat being the parent directory.

> along with a license file and executable within the subdirectory

it's not clear to me which directory "the subdirectory" is referring to.

I think the layout below makes it clear what `.nonmem_dir` should be, and this first paragraph is more likely to cause confusion.